webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/ComplianceLevel.scala

```diff
@@ -93,6 +93,44 @@ final case class CompliancePercent(
   val compliance = success + repaired + notApplicable + compliant + auditNotApplicable
 }
 
+final case class ComplianceSerializable(
+    applying:                   Option[Double],
+    successNotApplicable:       Option[Double],
+    successAlreadyOK:           Option[Double],
+    successRepaired:            Option[Double],
+    error:                      Option[Double],
+    auditCompliant:             Option[Double],
+    auditNonCompliant:          Option[Double],
+    auditError:                 Option[Double],
+    auditNotApplicable:         Option[Double],
+    unexpectedUnknownComponent: Option[Double],
+    unexpectedMissingComponent: Option[Double],
+    noReport:                   Option[Double],
+    reportsDisabled:            Option[Double],
+    badPolicyMode:              Option[Double]
+)
+
+object ComplianceSerializable {
+  def fromPercent(compliancePercent: CompliancePercent) = {
+    ComplianceSerializable(
+      if (compliancePercent.pending == 0) None else Some(compliancePercent.pending),
```

Comment:
you should have a private def to avoid duplicating that logic and avoid risking micopying the name in both place